### ISSUE TITLE
Is there a way to dynamically add routes?

### ISSUE THREAD

**Issue by tuhao1020** [kataras/iris#1732]
I need to serve a static directory which passed by users after the program starts, I tried `CreateRoutes` but it didn't work.
Is there any way to achieve this requirement?

**Comment by tuhao1020** [comment]
it works with below code
```go
app.Get("/mapping/{name}/{dir}", func(ctx iris.Context) {
  app.HandleDir("/static/" + ctx.Params().Get("name"), iris.Dir("../" + ctx.Params().Get("dir")), iris.DirOptions{
	  ShowList:          true,
	  SPA:               false,
  })
  app.RefreshRouter()
  ctx.Text("OK")
})
```

another question is: how can I delete the routes?  There is a method named `SetStatusOffline`, but it doesn't delete the route.

**Comment by tuhao1020** [comment]
@kataras Do you have time to look at my questions? 😃

**Comment by kataras** [comment]
Yes @tuhao1020 through the `AddRouteUnsafe`. Example can be found at the [configuration.go file](https://github.com/kataras/iris/blob/484787e5a6d1ee8f9aa151e138b2c1509003597e/configuration.go#L547-L550)

**Comment by tuhao1020** [comment]
@kataras now I need a function like  `RemoveRoute(path string)` 😄
